contrib/highlight.js/CHANGES.md

```diff
@@ -0,0 +1,1386 @@
+## Version 9.3.0
```

Comment:
I don't see how it hurts... But if the version number is somewhere else too, we can probably remove it.